ansible/roles/docker/templates/ubuntu1804.Dockerfile.j2

```diff
@@ -17,6 +17,8 @@ RUN apt-get update && apt-get dist-upgrade -y && apt-get install -y \
     ccache \
     g++ \
     gcc \
+    g++-8 \
+    gcc-8 \
```

Comment:
FWIW I'm not convinced the containers created by `ansible/roles/docker/templates/ubuntu1804.Dockerfile.j2` are actually being used in our CI. 
i.e. 
- https://ci.nodejs.org/computer/test-digitalocean-ubuntu1804_container-x64-1/builds
- https://ci.nodejs.org/computer/test-digitalocean-ubuntu1804_container-x64-2/builds
- https://ci.nodejs.org/computer/test-softlayer-ubuntu1804_container-x64-1/builds

are all empty (compare to an actual non-container e.g. https://ci.nodejs.org/computer/test-packetnet-ubuntu1804-x64-1/builds)

Reply:
yeah, probably not used, they were at one stage for some temporary testing work and they also exist as a way of experimenting with offloading VM tests to container tests, but other than that they're probably idling (which really only costs us the memory impact of the jenkins nodes)